### ISSUE TITLE
Feature Request Add new values for Input Autofire for fix some games

### ISSUE THREAD

**Issue by Augusto7743** [0ldsk00l/nestopia#289]
If using a high value for autofire some games will ignore input. Maximum value not is usable of any mode not being a problem, but if you select another value how example value 4, 5 or 6 not work for all games.
Example selecting a value 5 will has autofire working fine in Double Dragon 2, but Makai-Mura autofire not work. Selecting another value Makai-Mura has autofire and Double Dragon 2 has a slow autofire.
If possible create another values between mid of current values perhaps fix that problem.
Thanks for read my post.

**Comment by rdanbrook** [comment]
I just tested using Mega Man 4 on a real console with 4 turbo controllers to see how they line up with the settings. This is what I found:

NES Advantage: Dial to the max = 3
NES Max: 3
Turbo Tech Controller/Beeshu Zipper: Max = 4, Mid = 8
Turbo Card: 6

I didn't have the issues in Double Dragon 2 with any of the controllers on the real console, but was able to reproduce the problem in Nestopia.

The way this setting works is to pulse the turbo button every N frames. That means 2 is every other frame, 3 is every 3 frames, and so forth. There is no way to make it more granular than this unfortunately, due to the way the system is emulated.

I will try to figure out why there are some issues with turbo. Another problem I found is in Crystalis, when running while slashing the sword.
